components/02-elements/form/checkbox/checkbox.hbs

```diff
@@ -1,9 +1,12 @@
 <div class="checkbox {{ class }}">
-    <input type="checkbox" id="{{ id }}" class="checkbox__field">
-    <svg class="checkbox__icon">
+    <input type="checkbox"
+           id="{{ id }}"
+           class="checkbox__field {{ input.class }}"
```

Comment:
you can add a `name` attribute, recommended sources: https://webaim.org/techniques/forms/controls but it works fine :)

Reply:
done